# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccUtils.h
# Commit: d07307df1220
# Full Hash: d07307df122035ff6ea8c68e3ccdf895d46cc2fd
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-11-10 09:23:09
# Regressor Bug: 1796733
# File Overlap Count: 1
# Description:
#   Bug 1796733 part 4: Unify nsAccUtils::GetSelectableContainer. r=morgan
#   
#   We need this to unify support for the implicit selected state on ARIA items.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160881
# ==============================================================================

diff -r e8d222e1fd87 -r d07307df1220 accessible/base/nsAccUtils.h
--- a/accessible/base/nsAccUtils.h	Thu Nov 10 04:19:27 2022 +0000
+++ b/accessible/base/nsAccUtils.h	Thu Nov 10 04:19:27 2022 +0000
@@ -91,6 +91,8 @@
    * @param  aAccessible  [in] the item accessible
    * @param  aState       [in] the state of the item accessible
    */
+  static Accessible* GetSelectableContainer(const Accessible* aAccessible,
+                                            uint64_t aState);
   static LocalAccessible* GetSelectableContainer(LocalAccessible* aAccessible,
                                                  uint64_t aState);
 
